demos/inputs/inputs.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>CSS Chassis - Inputs</title>
+	<meta name="description" content="Button skeleton for styling">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="../../dist/css/chassis.css">
+	<link rel="stylesheet" href="../demos.css">
+	<link rel="stylesheet" href="./inputs.css">
+	<!-- <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,400italic,700italic" rel="stylesheet"> -->
+</head>
+<body>
+	<h1>Inputs</h1>
+	<input type="text" placeholder="Can't change this really" disabled>
+	<input type="text" placeholder="Enter your Name">
+	<input type="email" placeholder="Enter your Email">
+	<input type="password" placeholder="Choose a password">
```

Comment:
This styling should be class base - not on element directly.  Again see http://view.css-chassis.com/master/demos/buttons.html